dev_guide/integrating_external_services.adoc

```diff
@@ -120,8 +118,11 @@ be] loopback (127.0.0.0/8), link-local (169.254.0.0/16), or link-local multicast
 value in the service defined in the previous step.
 
 [[mysql-define-service-using-fqdn]]
-==== Using FQDN
-Use an `ExternalName` service without defining any ports or endpoints.
+==== Using an External Domain Name
+
+`ExternalName` services do not have selectors, or any defined ports or
+endpoints. You can use an `ExternalName` service to assign traffic to an
+external service outside the cluster. 
```

Comment:
Should we just say:
  to a service outside the cluster.

Or:
  to an external service.